docs/fidesops/docs/guides/admin_ui.md

```diff
@@ -0,0 +1,101 @@
+# Administrator UI
+
+FidesOps provides a built-in control panel for managing privacy requests.
+
+![admin ui](../img/admin_ui.png)
+
+## Running Locally
+
+To get started, clone the [FidesOps repository](https://github.com/ethyca/fidesops/).
+
+### Creating the root user
+
+In the top-level `fidesops` directory, run `make user`.
+
+A series of prompts will walk you through creating a username and password. Passwords require 8 or more characters, upper and lowercase characters, a number, and a symbol. 
+
+This will create an Admin UI Root User that can be used to access the in-browser control panel, and additional [user endpoints](#managing-users).
+
+### Access the Control Panel
+
+From the root `fidesops` directory, run the following:
+``` sh
+    cd clients/admin-ui
+    npm install
+    npm run dev
+```
+
+This will navigate you to the `admin-ui` directory, install the necessary packages, and run the development environment.
+
+Navigate to `http://localhost:3000/`, and provide the Root User credentials you created in the previous step. For this user, the `email` field is simply the `user` that was created, and not a valid email address.
+
+!!! tip "Only one root user can be created. Subsequent users should be managed via the available [Users endpoints](#creating-a-user)."
```

Comment:
hmm, found a bug, if you log in as the root user, and then log out, since logging out deletes the client, then it deletes the flag that the particular user was the root.

I wonder if this root user should only be given `user:create` and `user:delete` scopes, since the point of creating that user in the shell was to be able to create a first user that could create other users.